CHANGELOG.md

```diff
@@ -39,14 +39,16 @@
 
 ### Release notes
 
-- alertmanager:
+- Alertmanager:
   - using`regex` field from the `Matcher` type is deprecated and it will be removed in a future version. See [CHANGELOG](https://github.com/prometheus-operator/prometheus-operator/blob/main/CHANGELOG.md#0570--2022-06-02)
```

Comment:
```suggestion
  - Using`regex` field from the `Matcher` type is deprecated and it will be removed in a future version. See [CHANGELOG](https://github.com/prometheus-operator/prometheus-operator/blob/main/CHANGELOG.md#0570--2022-06-02)
```